packages/quantic/package.json

```diff
@@ -54,6 +54,7 @@
     "@octokit/graphql-schema": "10.74.2",
     "@salesforce/eslint-config-lwc": "3.3.3",
     "@salesforce/sfdx-lwc-jest": "1.1.2",
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
Is this necessary in order to test the navigation?

Reply:
yes in order to be able to use the function : `toHaveAttribute` in the last test

Reply:
We don't need that.
If someday we choose to add it, we will but not through 1 PR that does something else.
It will be it's own task

Reply:
ok ill remove it